tests/drivers/adc/adc_api/boards/numaker_pfm_m467.overlay

```diff
@@ -0,0 +1,41 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/ {
+	zephyr,user {
+		io-channels = <&eadc0 0>, <&eadc0 2>;
+	};
+};
+
+&pinctrl {
+/* EVB's UNO Pin A4 & D0 for channel 0 & 2 --> PB0, PB2 */
+	eadc0_default: eadc0_default {
+		group0 {
+			pinmux = <PB0MFP_EADC0_CH0>,
+				 <PB2MFP_EADC0_CH2>;
+		};
+	};
+};
+
```

Comment:
Guess the comment want to be indented with the  node it refers to? plus it fits on 80 columns, no need to wrap it
```c
        /* EVB's UNO Pin A4 & D0 for channel 0 & 2 --> PB0, PB2 */              
        eadc0_default: eadc0_default {                                          
                group0 {                                                        
                        pinmux = <PB0MFP_EADC0_CH0>, <PB2MFP_EADC0_CH2>;        
                };                                                              
        };                                                                      
```

Reply:
Thanks and update it.